packages/helpers/src/utils.js

```diff
@@ -35,8 +35,25 @@ const listFiles = async (dir) => {
     .map(dirE => dirE.name);
 };
 
+/**
+ * Validate and retrieve the AsyncAPI info object.
+ *
+ * Throws an error if the provided AsyncAPI document has no `info` section.
+ *
+ * @param {object|null|undefined} info - The `info` object from an AsyncAPI document.
+ * @throws {Error} When `info` is `null` or `undefined`.
+ * @returns {object} The validated `info` object.
+ */
+const getInfo = (info) => {
+  if (!info) {
+    throw new Error('Provided AsyncAPI document doesn\'t contain info.');
```

Comment:
`getInfo(asyncapi.info())` - this is how this function is used, so it doesn't really bring much value, except for the error - but error is not accurate, as it talks about AsyncAPI not containing info, although the argument of function is info object, so might be that template developer makes an errror and do not pass the info object - which means we provide wrong error to the user

imho this is how this helper should be used, and wrap parser code: `getInfo(asyncapi)`